en/docs/learn/self-registration-and-account-confirmation.md

```diff
@@ -15,24 +15,9 @@ delete such accounts to manage resources better.
 
 The following sections walk you through configuring and trying out
 self-registration.
-
-!!! warning
-    From WSO2 IS 5.3.0 onwards there is a new implementation for identity
-    management features. The steps given below in this document follows the
-    new implementation, which is the **recommended approach** for self
-    registration.
-    
-    Alternatively, to see the steps on how to enable this identity
-    management feature using the **old implementation**, see [Self Sign Up
-    and Account Confirmation documentation in WSO2 IS
-    5.2.0](https://docs.wso2.com/display/IS520/Self-Registration+and+Account+Confirmation). The old
-    implementation has been retained within the WSO2 IS pack for backward
-    compatibility and can still be used if required.
-    
     
-!!! tip "Before you begin"
-    
-    Ensure that the " `         IdentityMgtEventListener        ` " with the
+!!! Warning 
+    If you have migrated from a previous IS version, ensure that the `IdentityMgtEventListener` with the
```

Comment:
Let's mention that, if there is no such entries for event.default_listener.xxx, in deployment.toml, no need to configure these

Reply:
Fixed: 34640c47b5c2100d1c6d914bcc9b6aeb406df775